### PR TITLE
fix: patient summary widget title icon clicks are now persistent

### PR DESCRIPTION
**Pull request by bradymiller**
fix: patient summary widget title icon clicks are now persistent

### CODE REVIEW

File: templates/patient/card/card_base.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <section class="{{ card_container_class_list|default(['card', 'mb-2'])|join(' ') }} {{ card_bg_color }} {{ card_text_color }}">
     <div class="card-body p-1">
         <h6 class="card-title mb-0 d-flex p-1 justify-content-between">
-            <button class="btn btn-sm btn-link" href="#" {% if forceAlwaysOpen != true %} data-toggle="collapse" data-target="#{{ id|attr }}" {% endif %} aria-expanded="true" aria-controls="{{ id|attr }}">{{ title|text }}</button>
+            <a class="text-left font-weight-bolder" href="#" {% if forceAlwaysOpen != true %} data-toggle="collapse" data-target="#{{ id|attr }}" {% endif %} aria-expanded="true" aria-controls="{{ id|attr }}">{{ title|text }}<i class="ml-1 fa fa-fw fa-sm fa-expand" data-target="#{{ id|attr }}"></i></a>
```
Comment (sjpadgett) [inline templates/patient/card/card_base.html.twig:4]: The font-weight-bolder will fix our title issue! Though don't know why not jus font-weight-bold but whomever styled may have a reason.

File: templates/patient/card/card_base.html.twig [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <section class="{{ card_container_class_list|default(['card', 'mb-2'])|join(' ') }} {{ card_bg_color }} {{ card_text_color }}">
```
Comment (sjpadgett) [inline templates/patient/card/card_base.html.twig:1]: this must have the card class for styles to work and as container for card body.
I have all this in my PR

Comment (bradymiller) [reply templates/patient/card/card_base.html.twig:1]: nice! Sounds like I can drop this PR then?

Comment (sjpadgett) [reply templates/patient/card/card_base.html.twig:1]: yep, The whole OemrUI logic got factored out when the user setting save post was changed, so for demographic that function will fail on save. The last refactor needed is container (demographics_ps_expand)
working it now

**Pull request closed without merging**